src/components/LoanCollections/KivaClassicSingleCategoryCarousel.vue

```diff
@@ -54,16 +73,17 @@ export default {
 	data() {
 		return {
 			selectedChannel: {},
-			showCarousel: false,
-			isUrgencyExpVersionShown: false
 		};
 	},
 	computed: {
 		loanQueryLimit() {
 			return this.loanDisplaySettings?.loanLimit ?? 1;
 		},
 		showViewMoreCard() {
-			return this.loanDisplaySettings?.showViewMoreCard ?? false;
+			if (this.loanDisplaySettings?.showViewMoreCard) {
+				return this.loanQueryLimit === this.selectedChannelLoanIds.length;
```

Comment:
Only show view more card if our loan length is equal to query limit, so we can assume there are more loans.